spec/graphql/directive_spec.rb

```diff
@@ -8,6 +8,7 @@
         # plain fields:
         skipFlavor: flavor @skip(if: true)
         dontSkipFlavor: flavor @skip(if: false)
+        dontSkipDontIncludeFlavor: flavor @skip(if: false), @include(if: false)
```

Comment:
Does this mean _don't_ skip and _don't_ include? 

Would `@skip(if: true) @include(if: true)` be a better test for this feature? Maybe I've got my wires crossed.